service_container.rst

```diff
@@ -1069,6 +1069,14 @@ application to production (e.g. in your continuous integration server):
 
     $ php bin/console lint:container
 
+    # optionally, you can force the resolution of environment variables;
```

Comment:
```suggestion
    # optionally, you can force to resolve the environment variables;
```
?

Reply:
The first one sounds more natural to me. Not sure if we can fully trust it, but I asked AI and it agrees on this 🤔